### ISSUE TITLE
Textbundle support

### ISSUE THREAD

**Issue by chriskrycho** [notaapp/issues#37]
Hey folks, delighted to see this. First thing I did was open my current weekly export of notes as Textbundles from Bear and see what it does, and I quickly noticed that while Nota is perfectly happy to treat Textbundle as a folder, it *doesn't* understand it at a more robust level than that. Optimally:

- it would treat a Textbundle as a file, not a folder (and support textpack, the compressed format, as well) 
- accordingly wiki links would work with Textbundles as they do with plain-text files on disk

Again, excited to see this!

**Comment by astoilkov** [comment]
Definitely agree. We should have better support for Text Bundle.

Would you need the Text Bundle support if Nota had an _Import Notes from Bear_ command?

Thanks!

**Comment by chriskrycho** [comment]
I would! I actually very much like having TextBundle on disk as my source of truth for everything—it lets me keep richer content with my Markdown source, but having the benefits of flat file organization. (Currently, *Bear* is my source of truth, but I still get good use out of the files on disk that way.)

**Comment by astoilkov** [comment]
Makes sense. I like this thinking behind Text Bundle. This is definitely something we will soon take a look at.
